KubeArmor/core/kubeUpdate.go

```diff
@@ -1515,3 +1515,123 @@ func (dm *KubeArmorDaemon) WatchHostSecurityPolicies() {
 // add maps for macros and audit policies in KubeArmorDaemon structure
 // add watcher functions for KubeArmorMacro and KubeArmorAuditPolicy
 // similar to SecurityPolicies, update event auditor with given macros and audit policies
+
+func waitCustomResourceDefinition(name string) {
+	for crdNotLoaded := true; crdNotLoaded; {
+		if !K8s.CheckCustomResourceDefinition(name) {
```

Comment:
I removed ApplyCustomResourceDefinitions in the master branch; since I merged our CRDs into the deployment YAML file.

Reply:
ApplyCustomResourceDefinitions removed from code